src/components/Chatting/MessageForm/MessageForm.tsx

```diff
@@ -9,8 +9,9 @@ import isVirtualKeyboardDetectedAtom from "recoil/isVirtualKeyboardDetectedAtom"
 type MessageFormProps = {
   isSideChat: boolean;
   showNewMessage: boolean;
-  handleSendMessage: (message: string) => void;
+  handleSendMessage: (message: string) => boolean;
   handleSendImage: (image: File) => void;
+  handleSendAccount: (account: string) => void;
```

Comment:
`handleSendAccount` type이 `(account: string) => boolean;` 인것 같습니다

```suggestion
  handleSendAccount: (account: string) => boolean;
```